api/server/server.go

```diff
@@ -167,6 +169,11 @@ func NewFromEnv(ctx context.Context, opts ...ServerOption) *Server {
 	opts = append(opts, WithNodeCert(getEnv(EnvCert, "")))
 	opts = append(opts, WithNodeCertKey(getEnv(EnvCertKey, "")))
 	opts = append(opts, WithNodeCertAuthority(getEnv(EnvCertAuth, "")))
+	ridProvider := &RIDProvider{
+		HeaderName:   getEnv(EnvRidHeader, "fn-request-id"),
```

Comment:
in `agent/runner_client` we're using `fn_request_id` from `common.RIDContextKey()` but this is `fn-request-id` - am not sure if we want to be consistent, pointing this out.

Reply:
good catch, thanks.